src/styles/_variables.scss

```diff
@@ -19,3 +19,6 @@ $dark-bg-primary: $nord0;
 $dark-bg-secondary: $nord1;
 $dark-bg-secondary: $nord3;
 $dark-text-primary: $nord6;
+
+// Transitions
+$default-transition: 0.3s ease-in-out;
```

Comment:
Can the duration (e.g. 0.3s, 0.5s) have a dedicated enum of costants? It will be easier to maintain